e2e/api/testingSupport.js

```diff
@@ -23,11 +23,11 @@ module.exports =  {
           if (response.incidentMessage) {
             incidentMessage = response.incidentMessage;
           } else if (businessProcess.status !== 'FINISHED') {
-            throw new Error(`Ongoing business process: ${businessProcess.camundaEvent}, status: ${businessProcess.status},`
+            throw new Error(`Ongoing business process: ${businessProcess.camundaEvent}, case id: ${caseId}, status: ${businessProcess.status},`
               + ` process instance: ${businessProcess.processInstanceId}, last finished activity: ${businessProcess.activityId}`);
           }
       });
-    });
+    }, 300, 1000);
```

Comment:
What's 300 and 1000 here?

Reply:
number of retries and retry timeout in ms

Reply:
would it make sense to have these as constants with names rather than numbers?

Reply:
oh yeah for readability 👍

Reply:
done